docs/resources/loadbalancer.md

```diff
@@ -9,7 +9,7 @@ description: |-
 
 # ionoscloud_loadbalancer
 
-Manages a Load Balancer on IonosCloud.
+Manages a [Load Balancer](https://docs.ionos.com/cloud/network-services/application-load-balancer/overview) on IonosCloud.
```

Comment:
```suggestion
Manages a Load Balancer on IonosCloud.
```

Reply:
Load Balancer is different from Application Load Balancer. No URL from docs.ionos.com here